### DISCUSSION TITLE
[Feature] Save Metadata to XMP sidecar file

### DISCUSSION THREAD

**Discussion by antgiant** [immich-app/immich#3796] [Feature Request]
### The feature

I am requesting that Immich provide an option to store its metadata in XMP sidecar files in addition to its default database. This allows for other applications to read the metadata and if #3794  is implemented even allows full import of photo metadata from other applications. This is discussed in quite a few other issues (for instance [here](https://github.com/immich-app/immich/discussions/1692#discussioncomment-5984136) and [here](https://github.com/immich-app/immich/issues/294#issuecomment-1402478248)) but I could not find a request for it. Hence this one :-)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Metadata edits have now worked like this for a while
